demo.mk

```
@@ -53,6 +54,19 @@ demo-infra-app: demo-setup-kind ## Deploy Postgres for the demo
 		-var k8s_config_context=$(K8S_CLUSTER_CONTEXT) \
 		$(EXTRA_VARS) || exit 1 \
 
+.PHONY: demo-infra-app-plan
+demo-infra-app-plan: demo-setup-kind ## Deploy Postgres for the demo
+	@mkdir -p $(TF_APP_STATE_DIR)
+	rm -f $(TF_APP_STATE_DIR)/*.tf
+	cp $(DEMO_ROOT)/infra/app/*.tf $(TF_APP_STATE_DIR)/.
+	$(TERRAFORM) -chdir=$(TF_APP_STATE_DIR) init -upgrade
+	$(TERRAFORM) -chdir=$(TF_APP_STATE_DIR) plan \
```

```suggestion
	$(TERRAFORM) -chdir=$(TF_APP_STATE_DIR) apply \
```

I think you meant apply here?

---

Pretty sure it needs to be plan, as this is a new make target:
```
demo-infra-app-plan
```

We already have apply in the `demo-infra-app` target

---

Ah I forgot about that target, makes sense, thanks!